openapi-codegen/core/src/main/scala/sttp/tapir/codegen/openapi/models/RenderableModels.scala

```diff
@@ -0,0 +1,141 @@
+package sttp.tapir.codegen.openapi.models
+
+import sttp.tapir.codegen.openapi.models.OpenapiSchemaType.{
+  OpenapiSchemaArray,
+  OpenapiSchemaBinary,
+  OpenapiSchemaBoolean,
+  OpenapiSchemaDateTime,
+  OpenapiSchemaDouble,
+  OpenapiSchemaEnum,
+  OpenapiSchemaFloat,
+  OpenapiSchemaInt,
+  OpenapiSchemaLong,
+  OpenapiSchemaMap,
+  OpenapiSchemaObject,
+  OpenapiSchemaRef,
+  OpenapiSchemaString,
+  OpenapiSchemaUUID
+}
+
+// A value that can be written as a scala literal
+sealed trait RenderableValue {
```

Comment:
There's actually no need for this representation... could be achieved just by parsing to circe's Json repr and folding over the cases.